### ISSUE TITLE
Accuracy decreases during training

### ISSUE THREAD

**Issue by linxiangyulxy** [YantaoShen/kpm_rw_person_reid#9]
Hi, @YantaoShen , thanks for your great work!

I download your pretrained models on  CUHK03 dataset, and then run this commands:
`python examples/main.py -d cuhk03 -b 88 --features 2048 --alpha 0.95 --grp-num 2 --lr 0.000001 --ss 10 --epochs 10 --dropout 0.8 --combine-trainval --weight-decay 0 --retrain examples/logs/cuhk03-pretrained/model_best.pth.tar --logs-dir examples/logs/cuhk03-final-model`

But I am getting lower accuracy during training.

Could you please let me know how to solve this problem? Thank you in advance!

**Comment by YantaoShen** [comment]
Hi there, 

Thanks for your comments, I guess it is due to overfitting issue, could you provide more details like after how many epochs your model acc will drop?

Yantao
